Resources/Private/Fusion/Backend/Component/HeadIcons.fusion

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file is part of the Neos.Neos.Ui package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+include: resource://Neos.Fusion/Private/Fusion/Root.fusion
```

Comment:
this is sadly not a good idea to use fusion imo. The fusion parser will / cannot optimize this and the whole neos.fusion stuff is reparsed. That has to be done unfortunately because fusion allows global paths that can be overridden.

Reply:
You're confirming my worst fears there :sweat_smile: